### PR TITLE
[OSDOCS-4808] Changing "region" to "aws_region" (within open/close tags) where appropriate.

### PR DESCRIPTION
**Pull request by mletalie**
[OSDOCS-4808] Changing "region" to "aws_region" (within open/close tags) where appropriate. 

Version(s):

4.12+
Issue:

https://issues.redhat.com/browse/OSDOCS-4808
Link to docs preview:

1) https://55545--docspreview.netlify.app/openshift-dedicated/latest/networking/configuring-cluster-wide-proxy.html#cluster-wide-proxy-prereqs_configuring-a-cluster-wide-proxy

2) https://55545--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_aws/installing-aws-vpc.html#installation-custom-aws-vpc-requirements_installing-aws-vpc

3) https://55545--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_aws/installing-aws-private.html#installation-configure-proxy_installing-aws-private

4) https://55545--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_aws/installing-restricted-networks-aws.html#installation-creating-aws-bootstrap_installing-restricted-networks-aws

5) https://55545--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_aws/installing-restricted-networks-aws.html#installation-creating-aws-control-plane_installing-restricted-networks-aws

6) https://55545--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_aws/installing-aws-vpc.html#installation-custom-aws-vpc-requirements_installing-aws-vpc

7) https://55545--docspreview.netlify.app/openshift-rosa/latest/rosa_planning/rosa-sts-setting-up-environment.html#rosa-sts-setting-up-environment_rosa-sts-setting-up-environment

8) Please note that for https://github.com/openshift/openshift-docs/pull/55545/files#diff-5af7bc08d4ce71014254c1b9b1c6c20cbedf8a5fb25e4009b739474b28b04bdc, there will be no preview as the affected module is no longer in current documentation, 

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by mletalie** [comment]
Hello @yuwang-RH, could you please review this PR? Thanks!

**Comment by yuwang-RH** [comment]
LGTM~

File: modules/capi-yaml-infrastructure-gcp.adoc [hunk 1]
```diff
@@ -18,7 +18,7 @@ spec:
   network:
     name: <cluster_name>-network <2>
   project: <project> <3>
-  region: <region> <4>
+  region: <aws_region> <4>
```
Comment (aldiazRH) [inline modules/capi-yaml-infrastructure-gcp.adoc:21]: Does this region change apply to this page, since it's a GCP page?

**Review by aldiazRH — COMMENTED** [review]
Hey Mark, just a few things. Also, the PR name should be in this format, for future reference: 

PR title format: [GH#<gh-issue-id>][BZ#<bz-issue-id>][OCPBUGS#<jira-issue-id>][OSDOCS-<jira-issue-id>]: <short-description-of-the-pr>

So "[OSDOCS-4808] Changing region names" throughout or something to that effect.

**Comment by jeana-redhat** [comment]
@mletalie when you update the title according to convention, can you also please add direct preview links to the changes you have made here. Some of these are a bit hard to understand the context of without seeing them in the docs.

File: modules/capi-yaml-infrastructure-gcp.adoc (hunk 1 above)
Comment (mletalie) [reply modules/capi-yaml-infrastructure-gcp.adoc]: Hello @aldiazRH,
    Thanks for catching this issue. I reverted the change as it should not be included IMO. ML

**Review by jeana-redhat — APPROVED** [review]
LGTM

/remove-label peer-review-in-progress
/remove-label peer-review-needed
/label peer-review-done

**Pull request merged**